package.json

```diff
@@ -4,6 +4,7 @@
   "useWorkspaces": true,
   "workspaces": [
     "packages/base64",
+    "packages/cjs-module-analyzer",
```

Comment:
Again, why not `"packages/*"`?

Reply:
I changed this to `packages/*` in Agoric SDK one time and someone reverted it back. I don’t recall what it broke, but I believe some tool takes this list literally.